taxi/urls.py

```diff
@@ -53,11 +53,9 @@
     ),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
```

Comment:
Verify that the URL pattern for `taxi:driver-list` is correctly defined and points to the appropriate view.